hud-data/berryhud.json

```diff
@@ -1,12 +1,57 @@
 {
   "name": "BerryHUD",
-  "author": "hexereii",
+  "author": "heks",
   "social": {
-    "album": "https://imgur.com/a/ldt9NXv"
+    "steam_profile": "76561198041335138",
+    "steam_group": "",
+    "twitter": "hekstf",
+    "discord": "XQBfQtEE8E",
+    "album": "https://imgur.com/a/Q7QQwY2"
   },
+  "flags": [
+  "menus", 
+  "scoreboards", 
+  "crosshairs", 
+  "minmode",
+  "materials", 
+  "customization"
+  ]
+  "traits": {
+    "os": [
+	"windows", 
+	"mac", 
+	"linux"
+	],
+    "gamemode": [
+	"tournament", 
+	"comp", 
+	"cp", 
+	"halloween", 
+	"koth",
+	"mvm", 
+	"pass", 
+	"pl"
+	],
+    "res": 
+	[
+	"16/9"
+	],
+    "key": 
+	[
+	"value"
+	]
```

Comment:
you can delete these lines, and then remove the comma after the bracket on line 38